features/gateway_injection.md

```diff
@@ -0,0 +1,146 @@
+# Feature
+
+**Feature name:** Gateway Injection
+
+**Primary lead(s):** @howardjohn
+
+**Short description:** Injecting gateway deployments to enabled minimal configuration and simplified operations.
+
+---
+
+## Experimental
+
+### Requirements:
+
+- [ ] [User stories](insert_your_link_here) reviewed in a work group meeting.
+
+- [x] [RFC Authored] - [Injector V2](https://docs.google.com/document/d/1Rmp9B3DDypgMCau-YAqidx_r3qvKLZj6jUX-t22zj84/).
+
+- [x] [Documentation](https://preliminary.istio.io/latest/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental) and [here](https://preliminary.istio.io/latest/docs/setup/additional-setup/external-controlplane/) for enabling and using the feature.
+
+- [x] [Feedback plan](insert_your_link_here). N/A, we already have feedback from users that this is desired and useful.
+
+- [x] Disabled by default.
+
+- [x] No impact on performance when the feature is disabled.
+
+---
+
+## Alpha
+
+### Requirements:
+
+**Design**
+
+- [x] RFC has been approved describing the intention of the feature as well as the user stories behind the feature.
+
+**Config**
+
+- [x] Explicit user action is required to enable this feature (e.g. a config field, config resource, or installation action).
+
+> [Documentation](https://preliminary.istio.io/latest/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental)
+
+**Docs**
+
+- [x] Reference docs are published to preliminary.istio.io or the Istio wiki.
+  
+There are no reference docs required for this feature.
```

Comment:
I think there is a reference doc for the creation of custom templates you could link to. Also link to reference doc for the templates label.